editor/scene_tree_editor.cpp

```diff
@@ -295,6 +304,9 @@ bool SceneTreeEditor::_add_nodes(Node *p_node, TreeItem *p_parent) {
 		Ref<Script> script = p_node->get_script();
 		if (!script.is_null()) {
 			item->add_button(0, get_icon("Script", "EditorIcons"), BUTTON_SCRIPT, false, TTR("Open Script:") + " " + script->get_path());
+			if (EditorNode::get_singleton()->get_object_custom_type_base(p_node) == script) {
+				item->set_button_color(0, item->get_button_count(0) - 1, Color(1, 1, 1, 0.5));
```

Comment:
I would personally make it more transparent than 50%, since it's useful to know if you look closely but it's not something you usually need to see at a quick glance or when looking through your hierarchy.

Reply:
I initially made it disabled (and unclickable), but then decided to only use the disabled style (which is 50% transparent) with a working button.